src/inmanta/env.py

```diff
@@ -566,24 +566,38 @@ class PythonEnvironment:
     Inmanta product packages don't change.
     """
 
+    _invalid_chars_in_path_re = re.compile(r"[ \"&]")
```

Comment:
I can see that having a double quote can cause issues. But I am failing to see why we would also reject the whitespace and ampersand characters? Could you explain?

Reply:
Whitespace and ampersand are not causing issues after all. Added `$` and backtick char + some docstring
Full slack discussion [here](https://inmanta.slack.com/archives/CKRF0C8R3/p1699604351677279) for reference.